### PR TITLE
Actually use our preloader get function. Big speed boost switching tabs

### PR DESCRIPTION
**Pull request by finben**

### CODE REVIEW

File: Podfile.lock [hunk 1]
```diff
@@ -84,14 +84,14 @@ PODS:
...
-  - Realm (4.4.1):
-    - Realm/Headers (= 4.4.1)
-  - Realm/Headers (4.4.1)
-  - RealmSwift (4.4.1):
-    - Realm (= 4.4.1)
+  - Realm (5.0.0):
+    - Realm/Headers (= 5.0.0)
+  - Realm/Headers (5.0.0)
+  - RealmSwift (5.0.0):
+    - Realm (= 5.0.0)
```
Comment (skywalkerdude) [inline Podfile.lock:94]: Is there a reason you had to run a pod update? I'm not against it, just curious why you did it.

File: Hymns/Common/PDFViewer.swift [hunk 2]
```diff
@@ -3,7 +3,7 @@ import SwiftUI
 import PDFKit
 
 struct PDFViewer: UIViewRepresentable {
-
+    private let preloader: PDFLoader = Resolver.resolve()
```
Comment (skywalkerdude) [inline Hymns/Common/PDFViewer.swift:6]: lol, didn't realize we weren't using it

File: Podfile.lock (hunk 1 above)
Comment (finben) [reply Podfile.lock:94]: I kept getting that realm error when I was running the project. This fixed it on my local by upgrading our realm version.

Comment (skywalkerdude) [reply Podfile.lock:94]: Could you be more specific on what "that realm error" is?

**Pull request merged**